### PR TITLE
osdc/Objecter: resend pg commands on interval change

### PR DESCRIPTION
**Pull request by athanatos**

### CODE REVIEW

**Comment by athanatos** [comment]
Trying to get make check to run.

**Comment by athanatos** [comment]
retest this please

**Comment by athanatos** [comment]
http://pulpito.ceph.com/samuelj-2017-01-12_07:59:52-rados-wip-sam-testing---basic-smithi/
http://pulpito.ceph.com/samuelj-2017-01-12_16:56:54-rados-wip-sam-testing---basic-smithi/

File: src/osdc/Objecter.cc [hunk 1]
```diff
@@ -1296,7 +1296,7 @@ void Objecter::handle_osd_map(MOSDMap *m)
        p != need_resend_linger.end(); ++p) {
     LingerOp *op = *p;
     if (!op->session) {
-      _calc_target(&op->target, &op->last_force_resend);
+      _calc_target(&op->target);
```
Comment (liewegas) [inline src/osdc/Objecter.cc]: this hunk goes in the previous patch?

Comment (athanatos) [inline src/osdc/Objecter.cc]: ah, yeah

**Pull request merged**